components/NewsGrid.vue

```diff
@@ -0,0 +1,45 @@
+
+<template>
+	<div class="news-grid">
+		<div v-for="row in groupedList" :key="row.id" class="row">
+			<div v-for="x in row" :key="x.id" class="api-content">
+				<NewsItem
+					v-if="row.length>1"
```

Comment:
For readability, it's best to leave space around operators. Also see other comment about removing the condition.

Reply:
I seem to have lost the comment about the conditional, so I'll add it here. It seems this conditional is setup so that an inline style can be added in some cases. I believe we want to avoid inline styles. I suggest for now to just remove the condition and the second `<NewsItem>`. We can explore ways to handle this with CSS when we get to the styling.

Reply:
These changes have been implemented

Reply:
> For readability, it's best to leave space around operators. Also see other comment about removing the condition.

After adding spaces around operators I get the following error:

`3:12  warning  Unexpected spaces found around equal signs  vue/no-spaces-around-equal-signs-in-attribute`

Reply:
Ref: https://github.com/vuejs/eslint-plugin-vue/blob/master/docs/rules/no-spaces-around-equal-signs-in-attribute.md

Reply:
I should clarify, when I talk about operators I specifically meant `row.length>1` where the ">" has no whitespace. I didn't mean the "=" for the attribute assignments.